packages/xo-server-sdn-controller/package.json

```diff
@@ -28,6 +28,7 @@
   },
   "dependencies": {
     "@xen-orchestra/log": "^0.1.4",
+    "Math": "^0.0.1-security",
```

Comment:
`Math` is a global var.

Reply:
Yes i realised it when I used `max` in another PR, I was about to remove it. :)